src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/RedirectController.cs

```diff
@@ -40,9 +41,11 @@ public RedirectController(IOptions<GeneralSettings> settings)
         [HttpGet]
```

Comment:
Should the param description be changed here to specify that expected input is a base64  encoded string? 

Also missing some error handling here. I guess line 46 will throw an exception if the url is not base64 encoded, maybe check for this and return a bad request before attempting to decode the url?

Reply:
Agreed, fixing